swift/Package.swift

```diff
@@ -18,5 +18,8 @@ let package = Package(
         .target(
             name: "FlatBuffers",
             dependencies: []),
+        .testTarget(
```

Comment:
Can you please move the test case to the following dir `tests/FlatBuffers.Test.Swift` instead of creating it again in the swift dir?

Reply:
Oh, sorry, I have moved it.